src/data/polynomial/eval.lean

```diff
@@ -222,7 +222,8 @@ lemma eval_sum (p : polynomial R) (f : ℕ → R → polynomial R) (x : R) :
   (p.sum f).eval x = p.sum (λ n a, (f n a).eval x) :=
 eval₂_sum _ _ _ _
 
-
+lemma eval_finset_sum (s : finset ι) (g : ι → polynomial R) (x : R) :
```

Comment:
```suggestion
lemma eval_sum (s : finset ι) (g : ι → polynomial R) (x : R) :
```

Reply:
Hmm, I I think that `eval₂_finset_sum` isn't named as expected either. I would have expected `eval₂_sum` there.
Would you mind fixing that as well?

Reply:
There is already an `eval_sum` and `eval₂_sum`, so I think we have to stick with `eval_finset_sum` and `eval₂_finset_sum`

Reply:
Hmmm... I guess someone should do a general cleanup of the polynomial API... Don't worry about this for now.